### PR TITLE
chore(cli): include app version in app show

### PR DESCRIPTION
**Pull request by iamhopaul123**
This PR adds the application version to the app show human string as below
```
About

  Name              my-app
  Version           v0.0.0 (latest: v1.0.1)
  URI               example.com

Environments

  Name              AccountID           Region
  ----              ---------           ------
  test              123456789           us-west-2
  prod              123456789           us-west-1

Services

  Name              Type
  ----              ----
  my-svc            lb-web-svc

Pipelines

  Name
  ----
  pipeline1
  pipeline2
```
Also, it fixes a bug within the CFN pkg that doesn't include the app name when retrieving info from the last deployed app stackset.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: internal/pkg/cli/app_upgrade.go [hunk 1]
```diff
@@ -60,11 +66,31 @@ func newAppUpgradeOpts(vars appUpgradeVars) (*appUpgradeOpts, error) {
 		store:          store,
 		identity:       identity.New(sess),
 		prog:           termprogress.NewSpinner(log.DiagnosticWriter),
+		sel:            selector.NewSelect(prompt.New(), store),
 		versionGetter:  d,
 		upgrader:       cloudformation.New(sess),
 	}, nil
 }
 
+// Validate is a no-op for this command.
```
Comment (SoManyHs) [inline internal/pkg/cli/app_upgrade.go]: ...bad comment copy pasta?
```suggestion
```

File: internal/pkg/cli/app_upgrade.go [hunk 2]
```diff
@@ -60,11 +66,31 @@ func newAppUpgradeOpts(vars appUpgradeVars) (*appUpgradeOpts, error) {
...
+	if o.name != "" {
+		_, err := o.store.GetApplication(o.name)
+		if err != nil {
+			return fmt.Errorf("get application %s: %w", o.name, err)
+		}
+	}
+	return nil
+}
+
+// Ask prompts is a no-op for this command.
```
Comment (SoManyHs) [inline internal/pkg/cli/app_upgrade.go]:
```suggestion
```

**Review by SoManyHs — APPROVED** [review]

File: internal/pkg/describe/app.go [hunk 3]
```diff
@@ -46,6 +47,7 @@ func (a *App) HumanString() string {
 	fmt.Fprint(writer, color.Bold.Sprint("About\n\n"))
 	writer.Flush()
 	fmt.Fprintf(writer, "  %s\t%s\n", "Name", a.Name)
+	fmt.Fprintf(writer, "  %s\t%s (latest: %s)\n", "Version", a.Version, deploy.LatestAppTemplateVersion)
```
Comment (efekarakus) [inline internal/pkg/describe/app.go]: what do you think of coloring if `a.Version != deploy.LatestAppTemplateVersion`?

```go
availableVersion := "\n"
if deploy.LatestAppTemplateVersion != a.Version {
    availableVersion = color.Yellow.Sprintf("(latest available: %s)\n", deploy.LatestAppTemplateVersion)
}
```

**Review by efekarakus — APPROVED** [review]
+1 to the stale comments :D

File: internal/pkg/describe/app.go (hunk 3 above)
Comment (iamhopaul123) [reply internal/pkg/describe/app.go]: oh that's a good idea

**Pull request merged**